cron.sh

```diff
@@ -3,18 +3,18 @@ set -euo pipefail # See http://redsymbol.net/articles/unofficial-bash-strict-mod
 IFS=$'\n\t'
 
 # Execute via crontab by hduser@weywot1:
-# 30 6 * * * ssh sol@quaoar1 "cd /home/sol/git/mabxml-elasticsearch ; bash -x cron.sh >> logs/cron.sh.log 2>&1"
+# 30 5 * * * ssh sol@quaoar1 "cd /home/sol/git/mabxml-elasticsearch ; bash -x cron.sh >> logs/cron.sh.log 2>&1"
 
+TARGET_PATH=updates
 # Determine the latest update file and store it locally:
-updates=http://dataproxy.lobid.org/alephxml/export/update/
 date=$(date "+%Y%m%d")
-updateFile=$(curl --fail $updates | grep 'tar.gz' | cut -d '"' -f2 | grep $date)
-cd updates ; wget -nv $updates$updateFile ; cd ..
-# first, copy to the full data directory , see hbz/lobid#310.
-cp updates/* /files/open_data/open/DE-605/mabxml/
+date_yesterday=$(date --date yesterday "+%Y%m%d")
 
+cd $TARGET_PATH 
+wget -nv http://lobid.org/download/dumps/DE-605/mabxml/DE-605-aleph-update-marcxchange-$date_yesterday-$date.tar.gz
+cd ..
 # Run the transformation with the latest file (and possibly unprocessed previous files):
 curl --fail -XPOST "http://localhost:7300/hbz01/transform?dir=/home/sol/git/mabxml-elasticsearch/updates/&suffix=gz&cluster=quaoar1&hostname=193.30.112.170&index=hbz01" >> logs/processMabxml.sh.$date.log 2>&1
```

Comment:
Maybe this would be an improvement. However, this has nothing to do with this issue, if I am not erring. I mean, this feature was lacking before this pull request, doesn't it?
Using a variable doesnt' imply necessarily a configurable program - there is no parameter used in this script (which, in contrast, _would_ imply a configurable program).
What do you think?